eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ComplexSettings/ComplexSettingValueResolver.php

```diff
@@ -33,7 +33,7 @@
  * );
  * ```
  */
-class ComplexSettingValueFactory
+class ComplexSettingValueResolver
```

Comment:
I'd almost be tempted to ask for an interface :wink: 

Suggestion: `StackComplexSettingValueResolver implements ComplexSettingValueResolver`